bin/collator/src/cli.rs

```diff
@@ -12,6 +12,47 @@ pub struct Cli {
     #[clap(flatten)]
     pub run: cumulus_client_cli::RunCmd,
 
+    /// Enable EVM tracing module on a non-authority node.
+    #[clap(
+        long,
+        conflicts_with = "collator",
+        conflicts_with = "validator",
+        value_delimiter = ','
+    )]
+    pub ethapi: Vec<EthApi>,
+
+    /// Number of concurrent tracing tasks. Meant to be shared by both "debug" and "trace" modules.
+    #[clap(long, default_value = "10")]
+    pub ethapi_max_permits: u32,
+
+    /// Maximum number of trace entries a single request of `trace_filter` is allowed to return.
+    /// A request asking for more or an unbounded one going over this limit will both return an
+    /// error.
+    #[clap(long, default_value = "500")]
+    pub ethapi_trace_max_count: u32,
+
+    /// Duration (in seconds) after which the cache of `trace_filter` for a given block will be
+    /// discarded.
+    #[clap(long, default_value = "300")]
+    pub ethapi_trace_cache_duration: u64,
+
+    /// Size in bytes of the LRU cache for block data.
+    #[clap(long, default_value = "300000000")]
+    pub eth_log_block_cache: usize,
+
+    /// Size in bytes of the LRU cache for transactions statuses data.
+    #[clap(long, default_value = "300000000")]
+    pub eth_statuses_cache: usize,
+
+    /// Size in bytes of data a raw tracing request is allowed to use.
+    /// Bound the size of memory, stack and storage data.
+    #[clap(long, default_value = "20000000")]
+    pub tracing_raw_max_memory_usage: usize,
+
+    /// Maximum number of logs in a query.
+    #[clap(long, default_value = "10000")]
+    pub max_past_logs: u32,
+
```

Comment:
Good docs page will be needed for this.

Reply:
Agree, parameters should be explained in details.